personal-website/content/about.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+    <!-- Header -->
+    <head>
+        <meta charset="UTF-8">
+        <script src="https://use.fontawesome.com/c0ac3b38ec.js"></script> <!--insert Linkedin and Github icons as classes --> 
+        <title>Saharai's Personal Portfolio WebPage</title>
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <link rel="stylesheet" type="text/css" href="/Users/ada/Developer/projects/personal-website/content/stylesheets/style_bio.css">
+    </head>
+    <!-- Body -->
+    <body>
+        <section id="banner">
+            <div class="banner-text">
+
+                <h1>Bio-she/her/ellx</h1>
+                <br>
+                <p>The incredible amount of overlap between Software Development and Neuroscience research motivated my journey into programming. In addition to analogous developmental processes in both fields, the opportunity to spearhead projects allowed me to learn about the significant role technology plays in advancing therapeutic findings. This increased awareness, coupled with prior exposure to programming in academic settings, rekindled my passion for innovation and problem solving. Once I officially became an Adie, it was a pleasant surprise to find out that the fast paced world of software engineering played to my strengths as a researcher and as someone who identifies with the neurodiverse community.
+
+                    I welcome challenges and am driven by my curiosity and dedication to pursue a career that allows for constant growth. My prior work experience has served me greatly in optimizing my time at ADA. Operating at a graduate student level daily, working under multiple guidelines, and strategic planning, has taught me to be resourceful and to adapt. From networking to debugging, my work ethic has served me well in my commitment to follow my curiosty and pursue my passion for learning. In addition to giving me access to a great curriculum, ADA has taught me how to feel comfortable with the uncomfortable, and that has been the key to my success and a great daily reminder. Hence, now I am confident in pursuing my interests in full-stack based work. I look forward to bringing my technical and transferrable skills into the workforce and take ownership of my accomplishments and contribute to team goals.</p>
+                <div class="banner-btn">
+                    <a href="/Users/ada/Developer/projects/personal-website/content/index.html"><span></span>Home</a>
```

Comment:
Same Comment as above about relative path. However, because `about` and `index` are in the same directory so there's no need to add the folder in the path.

```suggestion
                    <a href="index.html"><span></span>Home</a>
```